README.md

```diff
@@ -71,6 +73,26 @@ dataset
 }
 ```
 
+
+## Diagnose your data generated
+
+```python
+from fiddlecube import FiddleCube
+fc = FiddleCube(api_key="<api-key>")
+result = fc.diagnose([
+        {
+            "query": "What is the capital of France?",
+            "answer": "Paris",
+            "prompt": "You are an expert at answering hard questions.",
+            "context": [
+                "Paris is the capital of France."
+            ]
```

Comment:
let's add a better example - pick one of the wheat examples from above

Reply:
On it.

Reply:
Done